mk/compose.mk

```diff
@@ -29,7 +29,7 @@ compose-down: ## Shut down service  depdencies using podman(docker)-compose
 
 .PHONY: compose-clean ## Clear out data (dbs, files) for service dependencies
 compose-clean: compose-down
-	$(DOCKER) volume prune --force
+	$(DOCKER) volume prune --force --all
```

Comment:
sadly this flag isn't supported in Podman and when I checkout this PR and run `make compose-clean` it fails with this error:
```
podman volume prune --force --all
Error: unknown flag: --all
```
\
is it necessary to add this flag? 🤔

Reply:
unfortunately it seems either this flag or `--filter all=true` is necessary for docker to remove all anon and named volumes now, but neither are supported by podman 😞  added in a conditional check to get around this. should work for both now 🤞

Reply:
alrighty, `make compose-clean` works now, thanks! 🎉